iXBRLViewerPlugin/viewer/src/js/inspector.js

```diff
@@ -367,8 +389,17 @@ Inspector.prototype.update = function () {
     this.updateURLFragment();
 }
 
-Inspector.prototype.selectFact = function (id) {
+Inspector.prototype.selectFact = function (id, factIdSet) {
```

Comment:
This seems like an odd signature.  Add a second `selectFacts(ids)` and then change this one to call that one with `selectFact(id)`?

Reply:
The issue is that both id and factIdSet can be provided: the former is the specific fact to be selected, and the latter is a list of all facts that the click could correspond to (when you have nested facts).

As it stands, the selected fact will always be the first fact in the list, as this is 

I've attempted to clarify in dcb4524.

Reply:
That helped, thanks.